routes/auth.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");  // pentru a crea servere web și a gestiona rutele API.
+const router = express.Router();     // submodul ce ajută la organizarea și modularizarea rutelor aplicației
+const dotenv = require('dotenv');    // pentru  încărcarea variabilelor de mediu dintr-un fișier .env în aplicație
+const jwt = require('jsonwebtoken'); // pentru crearea și validarea tokenurilor JWT
+const sha256 = require('sha256');    // pentru criptarea și compararea parolelor
+const DB_URL = process.env.DB_URL;
```

Comment:
nu ai nevoie de db_url aici, nu?

Reply:
stiind ca dashboardul are un admin, nu cred ca este nevoie sa il pastram in BD, se poate in **.env**, dar sa fie encrypted
https://emn178.github.io/online-tools/sha256.html